build.gradle

```diff
@@ -79,10 +79,6 @@ dependencies {
     implementation name: 'jetty-all-9.4.0.v20161208-uber'
     implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.6'
 
-
-    implementation 'com.sparkjava:spark-core:2.8.0'
-    implementation 'com.sparkjava:spark-template-velocity:2.7.1'
-
```

Comment:
This change should be made in another branch.

Reply:
No it's not, the dependencies only being used by the value tuner (and now they are not...)